pkgs/top-level/all-packages.nix

```diff
@@ -1610,7 +1610,9 @@ with pkgs;
 
   ddrescue = callPackage ../tools/system/ddrescue { };
 
-  deluge = python2Packages.deluge; # Package should be moved out of python-packages.nix
+  deluge = callPackage ../applications/networking/p2p/deluge {
+    pythonPackages = python2Packages;
```

Comment:
Should all packages that are Python 2 or 3 only change to that convention?  There are a few other packages that do that, like beets, buildbot, and catfish.